bundles/org.openhab.binding.etherrain/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="etherrain" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- The EtherRain Thing Type -->
+    <thing-type id="etherrain">
+        <label>EtherRain</label>
+        <description>This is a stand alone EtherRain device that allows sprinkler control</description>
+
+        <channels>
+            <channel id="commandstatus" typeId="commandstatus" />
+            <channel id="operatingstatus" typeId="operatingstatus" />
+            <channel id="operatingresult" typeId="operatingresult" />
+            
+            <channel id="relayindex" typeId="relayindex" />                   
+            <channel id="rainsensor" typeId="rainsensor" />
+            
+            <channel id="execute" typeId="execute">
+                <label>Execute</label>
+                <description>Commands EtherRain device to begin watering program</description>
+            </channel>
+            <channel id="clear" typeId="execute">
+                <label>Clear</label>
+                <description>Clears the currently running program</description>
+            </channel>
+        </channels>
+
+        <config-description>
+            <parameter name="host" type="text" required="true">
+                <label>Host Name</label>
+                <description>The host name or IP address of the EtherRain Web API interface</description>
+                <context>network-address</context>
+            </parameter>
+            <parameter name="port" type="integer" required="true" min="1" max="65335">
```

Comment:
```suggestion
            <parameter name="port" type="integer" min="1" max="65335">
```
If you set a default, it is not required. required means "the user needs to set it for the thing to work".